web/server/api/endpoints/context.py

```diff
@@ -18,7 +18,7 @@ def get_api_context(
 ) -> models.Context:
     """Get the context"""
 
-    context.refresh()
+    context.load()
```

Comment:
I had another similar comment somewhere else, but is this still needed if the file watcher will reload the context automatically?